interfaces/matlab_experimental/Base/ThermoPhase.m

```diff
@@ -449,19 +459,7 @@ function delete(tp)
         %% ThermoPhase Utility Methods
 
         function display(tp)
-            % Display thermo properties
-
-            buflen = 0 - calllib(ctLib, 'thermo_report', tp.tpID, 0, '', 1);
-            aa = char(ones(1, buflen));
-            ptr = libpointer('cstring', aa);
-            [iok, bb] = calllib(ctLib, 'thermo_report', tp.tpID, buflen, ptr, 1);
-
-            if iok < 0
-                error(ctGetErr);
-            else
-                disp(bb);
-            end
-
+            ctFunc('thermo_print', tp.tpID, 1, 1);
```

Comment:
I'd suggest implementing this as `disp(tp.report)`

Reply:
The problem I see with that approach is `thermo_report` currently doesn't have a display threshold for mass and mole fractions. If there are many species in the input file the display could be inconveniently long. Should we modify it so it could accept that as an argument? Then we could maybe deprecate `thermo_print`. 

I made a mistake here as the threshold should be set to a lower value like 1e-4. Currently the threshold is set to 1.